Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -1624,43 +2084,366 @@ def zoom_list_messages_command(client, **args) -> CommandResults:
     )
 
 
+def get_channel_jid_from_context(channel_name: str = None, investigation_id=None):
```

Comment:
First of all, in this method, make sure that at least one of the parameters is provided.
Log if none is provided.